.go-version

```diff
@@ -1 +1 @@
-1.19.9
```

Comment:
😂 The newline has been added. Looks good!

Reply:
Yes, I saw it. Since it did not break anything, So I did not raise the comment:); otherwise I might need to raise the same comment each time when other contributors bump the go version.